nemo/lightning/__init__.py

```diff
@@ -82,7 +82,8 @@ def _is_slurm_interactive_mode():
 import warnings
 
 warnings.warn(
-    "nemo.lightning is deprecated and will be removed in the next major NeMo FW container release.",
+    "nemo.lightning is deprecated and will be removed in a future major NeMo FW container release. "
+    "Please refer to the new Megatron-Bridge repository: https://github.com/NVIDIA-NeMo/Megatron-Bridge",
```

Comment:
isn't this still used by speech? in that case we shouldn't have the deprecation warning here